extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/OtTracerPropagator.java

```diff
@@ -66,6 +69,15 @@ public static OtTracerPropagator getInstance() {
         spanContext.getTraceIdAsHexString().substring(TraceId.getHexLength() / 2));
     setter.set(carrier, SPAN_ID_HEADER, spanContext.getSpanIdAsHexString());
     setter.set(carrier, SAMPLED_HEADER, String.valueOf(spanContext.isSampled()));
+
```

Comment:
Leave a note mentioning that `Baggage` is only injected if there was a valid `SpanContext` - likewise for extraction ;)

Reply:
done